src/App.js

```diff
@@ -139,6 +137,12 @@ class App extends Component {
   }
 }
 
+const mapDispatchToProps = dispatch => ({
+  addNewSite: (newSiteObject) => {
```

Comment:
I think here you can use just following notation (without the arrow function):
```
addNewSite(newSiteObject) {
  dispatch(addSite(newSiteObject));
},
```
But it's up to you.